### PR TITLE
Add document version and update docId format

### LINKED ISSUE #2: View a specific anchor of a document
It should be possible to reference and query a specific version of a document. This is useful if you want an immutable link to an older version document and be sure that it won't change. 

```
/ceramic/<genesis-CID>?anchor=<anchor-CID>
```

### PR DESCRIPTION
**Pull request by oed**
Fix #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,16 +58,30 @@ To look up a document the *docId* is needed. Once a *docId* is known a node can
 
 In Ceramic, each document has a unique identifier (docId). The identifier is a string with the following format:
```
Comment (michaelsena) [inline README.md:59]: Maybe add something about the unique identifier being persistent and never changing

File: README.md [hunk 2]
```diff
@@ -58,16 +58,30 @@ To look up a document the *docId* is needed. Once a *docId* is known a node can
...
 ```
-/ceramic/bafyreihl3rizxqjkedmp7rdckrqd3kufwe5e7c6xejcoheqo7rp63idsva
+ceramic://bafyreihl3rizxqjkedmp7rdckrqd3kufwe5e7c6xejcoheqo7rp63idsva
 ```
 
+### Document versions
+Each time a document is [anchored on a blockchain](#blockchain-anchoring) with an *anchor record*, a new version of the document is created. Each version of the document can be refered to by using the following format:
+
+```
+ceramic://<CID-of-genesis-record>?version=<CID-of-anchor-record>
```
Comment (michaelsena) [inline README.md:81]: Are we now just using this format and not also numerical versions?

File: README.md (hunk 1 above)
Comment (oed) [reply README.md:59]: Good idea!

File: README.md (hunk 2 above)
Comment (oed) [reply README.md:81]: Yeah, no numerical versions for now. This way is more secure.

**Pull request merged**